src/Lachain.Core/RPC/HTTP/Web3/TransactionServiceWeb3.cs

```diff
@@ -502,6 +502,8 @@ public string Call(JObject opts, string? blockId)
                         var localInvocation = ContractEncoder.Encode("transfer(address,uint256)", destination, tx.Value);
                         var transferResult =
                             ContractInvoker.Invoke(ContractRegisterer.LatokenContract, transferContext, localInvocation, GasMetering.DefaultBlockGasLimit);
+                        if (transferResult.Status != ExecutionStatus.Ok)
```

Comment:
should we rollback the snapshot before returing?

Reply:
It wiil be rolled back automatically in StateManager.Release()